tests/SPR.tex

```diff
@@ -92,6 +92,7 @@ \subsection{OC instructions}
 \begin{itemize}
     \item Announce the placement of the robots
     \item choose the volunteers for the second part of the test, and clearly explain the procedure to them.
+    \item When the test is held outside the arena, announce (way)point the robot to leave to
```

Comment:
I don't get the syntax here. Instead of ```When the test is held outside the arena, announce (way)point the robot to leave to``` shouldn't be either one of

- ```When the test is held outside the arena, announce the leaving (way)point```
- ```When the test is held outside the arena, announce the (way)point through which the robot shall leave```